_data/ui/cron-scheduling.yml

```diff
@@ -0,0 +1,212 @@
+    url: *crontab
+    description: "Simple, web-based editor for cron expressions."
+
+# -------------------------- #
+#      Required Fields       #
+# -------------------------- #
+
+required-fields:
+  - name: "minute"
+    allowed-values: "* or 0-59"
+    special-characters: |
+      , -
+
+  - name: "hour"
+    allowed-values: "* or 0-23"
+    special-characters: |
+      , -
+
+  - name: "day of month"
+    allowed-values: "* or 1-31"
+    special-characters: |
+      , -
+
+  - name: "month"
+    allowed-values: "* or 1-12"
+    special-characters: |
+      , -
+
+  - name: "day of week"
+    allowed-values: "* or 0-6 (0 = Sunday)"
+    special-characters: |
+      , -
+
+
+# -------------------------- #
+#     Special characters     #
+# -------------------------- #
+
+special-characters:
+  - character: "*"
+    name: "asterisk"
+    allowed-in: "all"
+    description: |
+      Selects all values within a field.
+    example: |
+      - `*` in the `minute` field means _"every minute"_
+      - `*` in the `month` field means _"every month"_
+      - `*` in the `day of week` field means _"every day of the week"_
+
+  - character: ","
+    name: "comma"
+    allowed-in: "all"
+    description: |
+      Specifies a list of two or more values.
+    example: |
+      - `1,2,5` in the `month` field means _"the months January, February, and May"_
+      - `1,5` in the `day of week` field means _"the days Monday and Friday"_
+
+  - character: "-"
+    name: "hyphen"
+    allowed-in: "all"
+    description: |
+      Specifies a range of values.
+    example: |
+      - `5-8` in the `hour` field means _"the hours 5, 6, 7, and 8"_
+      - `1-3` in the `day of week` field means _"the days Monday, Tuesday, and Wednesday"_
+
+
+# -------------------------- #
+#     Expression examples    #
+# -------------------------- #
+
+examples:
+  - translation: "Run every day at midnight"
+    expression: |
+      ```
+      0 0 * * *
+      ```
+
+  - translation: "Run at midnight every day from Monday through Friday"
+    expression: |
+      ```
+      0 0 * * 1-5
+      ```
+
+  - translation: "Run once a week at 11:00PM on Sunday"
+    expression: |
+      ```
+      0 23 * * 0
+      ```
+
+  - translation: "Run every day at noon and midnight"
+    expression: |
+      ```
+      0 0,12 * * *
+      ```
+
+  - translation: "Run Tuesday through Thursday every week at 3:00AM"
+    expression: |
+      ```
+      0 3 * * 2-4
+      ```
+
+  - translation: "Run at midnight every Monday, Wednesday, and Friday"
+    expression: |
+      ```
+      0 0 * * 1,3,5
+      ```
+
+  - translation: "Run at minute 5, 6, 7, 8, 9, 10, and 30 every day in January"
+    expression: |
+      ```
+      5-10,30 * * 1 *
+      ```
+
+  - translation: "Run every first day of the month and every week on Wednesday"
+    expression: |
+      ```
+      * * 1 * 3
+      ```
+
+  - translation: "Run every 15th day of the month in June and every Thursday at 5:00PM"
```

Comment:
Similarly "Run at 5:00PM on every 15th day of the month in June and every Thursday"